### PR TITLE
HYDRA-812 : Add Unit Test for Maya Display Layers

### PR DESCRIPTION
**Pull request by roopavr-adsk**

### CODE REVIEW

File: test/lib/mayaUsd/render/mayaToHydra/MayaDisplayLayersTest/layerReferenceSelected.png
Comment (lanierd-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/MayaDisplayLayersTest/layerReferenceSelected.png:1]: Shouldn't we see wireframe on the cone as well ?

File: test/lib/mayaUsd/render/mayaToHydra/MayaDisplayLayersTest/layerTemplateSelected.png
Comment (lanierd-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/MayaDisplayLayersTest/layerTemplateSelected.png:1]: Same here, shouldn't the cone be red as well ?

File: test/lib/mayaUsd/render/mayaToHydra/MayaDisplayLayersTest/layerReferenceSelected.png
Comment (roopavr-adsk) [reply test/lib/mayaUsd/render/mayaToHydra/MayaDisplayLayersTest/layerReferenceSelected.png:1]: line 97 and 105, I am specifically selecting only the sphere. Or did you mean I should select the cone too?

File: test/lib/mayaUsd/render/mayaToHydra/MayaDisplayLayersTest/layerTemplateSelected.png
Comment (roopavr-adsk) [reply test/lib/mayaUsd/render/mayaToHydra/MayaDisplayLayersTest/layerTemplateSelected.png:1]: line 97 and 105, I am specifically selecting only the sphere. Or did you mean I should select the cone too?

Comment (lanierd-adsk) [reply test/lib/mayaUsd/render/mayaToHydra/MayaDisplayLayersTest/layerTemplateSelected.png:1]: No, it's fine, thanks.

**Review by lanierd-adsk — APPROVED** [review]

**Pull request merged**